#Expyriment 0.9.0 (Python 3.6.6), .xpe-file, coding: ISO8859-1
#date: Tue Aug 14 2018 09:48:06
#sha1: None
#modules: 
#display: size=(800, 600), window_mode=True, open_gl=2
#os: uname_result(system='Linux', node='L3460', release='4.4.0-109-generic', version='#132~14.04.1-Ubuntu SMP Tue Jan 9 21:46:42 UTC 2018', machine='x86_64', processor='x86_64')
Time,Type,Event,Value,Detail,Detail2
266,stdout,received,'LEVEL 1'
4272,stdout,received,'Block None: LEVEL 1\n    block factors: LEVEL = LEVEL 1\n    n trials: 5\n    trial IDs = [0, 1, 2, 3, 4]\n    trial factors: '
5718,design,log,
#design: #exp: Arithmatic test
#design: block_cnt,block_id,block_LEVEL,trial_cnt,trial_id
#design: 0,0,LEVEL 1,0,0
#design: 0,0,LEVEL 1,1,1
#design: 0,0,LEVEL 1,2,2
#design: 0,0,LEVEL 1,3,3
#design: 0,0,LEVEL 1,4,4
5718,design,logged,
5718,Experiment,started
6719,Stimulus,presented,9
7720,stderr,received,'Traceback (most recent call last):'
7720,stderr,received,'  File "arithmatictask_v0.py", line 59, in <module>'
7720,stderr,received,'    s1.presemt()'
7720,stderr,received,"AttributeError: 'TextLine' object has no attribute 'presemt'"
